### DISCUSSION TITLE
Not getting email on Order confirmation

### DISCUSSION THREAD

**Discussion by mvtdev1050** [vendurehq/vendure#2688] [Q&A]
I have used vendure for one of my project . I am getting all emails like after verification email, reset password . But the user and admin not getting emails for Order confirmation . 
I have passed 'defaultEmailHandlers' in handlers like this-
` handlers: [...defaultEmailHandlers,ContactRequestEventHandler],`
Do I need to setupor config anything in any other file?

**Comment by michaelbromley** [comment]
Hi,

Can you share the exact sequence of steps you take to reproduce this?
